bin/get-all-tasks

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+## This script get all tasks for SAPS interacting with Dispatcher backend.
+
+## checking args
+if [ $# -ne 3 ]
+then
+  echo "Usage: $0 USER_EMAIL USER_PASSWORD DISPATCHER_ADDRESS"
+  exit 1
+fi
+
+# args
+readonly USER_EMAIL=$1
+readonly USER_PASSWORD=$2
+readonly DISPATCHER_ADDRESS=$3
+
+ALL_TASKS=`curl --silent --request GET "${DISPATCHER_ADDRESS}/processings" --header "Content-Type: text/plain"  --header "userEmail: $USER_EMAIL"  --header "userPass: $USER_PASSWORD"`
+JSON_LENGTH=$((`echo $ALL_TASKS | jq 'length' $JSON_PATH` - 1))
```

Comment:
$JSON_PATH is not used by the jq command. It is empty.
Use $() instead backticks. Backticks have been deprecated.

Reply:
Done